services/auth/src/api.js

```diff
@@ -1060,6 +1060,9 @@ builder.declare({
     req.clientId(),
     req.scopes(),
   ]);
+  //setting headers as purposed by bug 161375
```

Comment:
Including a bug number in a comment is useful when a reader might otherwise wonder "why was this written??".  But in this case, at least once the patch is finished, the reasoning should be pretty clear from reading the code, so there won't be a need to reference the bug, and even a comment may feel somewhat redundant.